컴파일 -정상적으로 실행 됐으면 

런타임- 에러가 생길경우 catch 문으로 넘어가서 오류메시지 출력 후 printStackTrace(); 를 입력하면
출력스트림을 보여줌



Exception05 왜 이해가 안되지 ?

입셉션으로부터 상속을 받은 클래스를 생성가능
체크라는 메서드 만들고 이것을 위의 클래스로 떤진다

메인 실행문 입셉션 05 obj 만들고 

체크에 60이 들어감 이게 트루니까 위의 뜨루뉴가 실행되고
캐치문으로 넘어가서 
예외처리와 getMessage가 실행됨 ->InvalidException